README.md

```diff
@@ -7,3 +7,14 @@ devices in the cloud.
 The application is developed for and tested on Google Cloud Platform, but it's
 designed in a way that should allow its deployment in other Cloud solutions
 without much trouble.
+
+## Links for each component
+
+To know details, please refer to these documents below.
+
+- [Cloud Orchestrator](docs/cloud_orchestrator.md): A web service for hosting
```

Comment:
s: `for managing`

Reply:
https://github.com/google/cloud-android-orchestration/pull/321